<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordsWindow</class>
 <widget class="QWidget" name="RecordsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bt_return">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_print">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_recordTitle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>详单</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_db">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="theColorBar" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="lb_user_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_room_db">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>房间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_times_db">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>请求次数：</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
